app/src/main/java/org/simple/clinic/summary/teleconsultation/messagebuilder/TeleconsultationSmsMessageBuilder.kt

```diff
@@ -0,0 +1,19 @@
+package org.simple.clinic.summary.teleconsultation.messagebuilder
+
+import org.simple.clinic.summary.PatientTeleconsultationInfo
+import javax.inject.Inject
+
+class TeleconsultationSmsMessageBuilder @Inject constructor() : TeleconsultationMessageBuilder {
+
+  // TODO (SM): 2020-08-24 Change the deep link
+  override fun message(patientTeleconsultationInfo: PatientTeleconsultationInfo): String {
+    val message = StringBuilder("Teleconsult request")
+        .appendLine()
+        .appendLine()
+        .appendLine("Review patient & log a record:")
+        .appendLine("https://app.simple.org/patient/${patientTeleconsultationInfo.patientUuid}")
```

Comment:
I am assuming that the change to the teleconsult URL format will happen later.

Reply:
Yep, I am trying to figure out how to handle both the old and new deep links URLs.